src/js/graphql/gql/fragments.ts

```diff
@@ -160,7 +160,6 @@ export const QUERY_ALL_COUNTRIES = gql`
       areaName
       uuid
       totalClimbs
-      updatedAt
```

Comment:
Can you create a new branch for each PR so that we don't have old (already merged) changes?  Thanks!